uaclient/entitlements/esm.py

```diff
@@ -97,6 +97,10 @@ class ESMAppsEntitlement(ESMBaseEntitlement):
     description = messages.ESM_APPS_DESCRIPTION
     help_text = messages.ESM_APPS_HELP_TEXT
     repo_key_file = "ubuntu-pro-esm-apps.gpg"
+    snapshot_urls = [
+        "snapshot.apps-security.esm.ubuntu.com/apps/ubuntu/",
+        "snapshot.apps-updates.esm.ubuntu.com/apps/ubuntu/",
```

Comment:
I can't say I like those values hardcoded here, as changing them in the future will require another SRU of Pro to conclude, and more migrations.

Fetching it from APT should be possible but I don't know any good way of doing it without hacking the metadata from the release file, and that's a no.


Given the APT url for a service already comes from the contracts server, do you see any harm in getting that from CS as well? If the answer is no because of the time we have, could we please consider that for the near future, before other services have their own snapshots?